dependencyManagement/build.gradle.kts

```
@@ -27,36 +27,36 @@ data class DependencySet(val group: String, val version: String, val modules: Li
 val testSnapshots = rootProject.findProperty("testUpstreamSnapshots") == "true"
 
 // This is the version of the upstream instrumentation BOM
-val otelVersion = "2.11.0-adot1"
-val otelSnapshotVersion = "2.12.0"
+val otelVersion = "2.12.0"
```

let's keep the `-adot1` suffix
```suggestion
val otelVersion = "2.12.0-adot1"
```